### PR TITLE
cv 0.5.1

### PR DESCRIPTION
**Pull request by BestPig**
Adding cv (Coreutils Viewer) in version 0.4.1.

### CODE REVIEW

File: Library/Formula/cv.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require "formula"
+
+class Cv < Formula
+  homepage "https://github.com/BestPig/cv"
```
Comment (bfontaine) [inline Library/Formula/cv.rb]: Can’t you use [BestPig’s repo](https://github.com/BestPig/cv/releases/tag/v0.5.1) in the meantime? It’s linked in the upstream README as the official OSX port until they merge the support upstream.

Comment (mikemcquaid) [inline Library/Formula/cv.rb]: I'd probably rather wait for upstream to merge this.

**Comment by asparagui** [comment]
Still waiting on upstream.

**Comment by BestPig** [comment]
#40925 was merged, this issue can now be closed.

**Pull request closed without merging**